server/routes/forms.ts

```diff
@@ -0,0 +1,36 @@
+import { type RequestHandler, Router } from 'express'
+import asyncMiddleware from '../middleware/asyncMiddleware'
+import type { Services } from '../services'
+
+export default function routes({ dataFilterService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+
+  get('/', async (req, res) => {
+    const [productList] = await dataFilterService.getDropDownLists({
+      productName: '',
+      useFormattedName: true,
+    })
+
+    return res.render('pages/githubreporequestform', {
```

Comment:
suggestion: form names are generally camelCase in developer portal

Reply:
What Andy said